Harvest.Web/Views/Permissions/Index.cshtml

```diff
@@ -9,7 +9,7 @@
 <h1>Users with roles</h1>
     </div>
     <div class="col text-right">
- <a class="btn btn-sm btn-link" asp-action="Create"><i class="fas fa-plus fa-xs"/></i> Add new User/Role Font Awesome here</a>
+        <a class="btn btn-sm btn-link" asp-action="Create"><i class="fas fa-plus fa-xs"/></i> Add new User/Role Font Awesome here</a>

```

Comment:
Do we still need the "Font Awesome here" here?

Reply:
@sprucely  IIRC we are not loading FontAwesome on the CSHTML pages, just via react, so we still do need to figure out how to add font awesome here.  Likely we'll want to add it to the styles section on just the pages that it's needed and not the layout page so it is not double-loaded.